src/renderer/modules/components/RadioItem.tsx

```diff
@@ -47,11 +47,10 @@ export const RadioItem = (props: React.PropsWithChildren<RadioItemProps>): React
   return (
     <FormItem
       title={props.children}
-      style={{ marginBottom: 20 }}
+      style={{ marginBottom: 20, ...props.style }}
```

Comment:
We should use a different property for this so we can style it separately from the component itself (applies to other files as well)